pepper-apis/build-utils/build-support-images/README.md

```diff
@@ -0,0 +1,34 @@
+# Creating build and deployment-support Docker images
+
+## Overview
+Our builds and deploys are executed via CircleCi.
+
+We have two images used in our CircleCi configuration.
+
+The first one is there to support building and deploying the app and is built with `Dockerfile-build`.
+The second one is more focused on supporting running our suite of tests and is built with `Dockerfile-java-build`
+
+For convenience and to minimize external dependencies, we are also using one of these images to run `consul-template`
+to generate all the runtime and test configuration files.
+
+
+## Requirements
+On Mac computer we are using:
+`Docker version 20.10.14, build a224086`
+
+## Procedure
+
+``docker buildx build --platform linux/amd64,linux/arm64 -t broadinstitute/study-server-build:2022-06-01A  --build-arg GIT_SHA=`git rev-parse --verify HEAD` -f ./Dockerfile-build --push .
+``
+
+``docker buildx build --platform linux/amd64,linux/arm64 -t broadinstitute/study-server-build:java-2022-06-01A  --build-arg GIT_SHA=`git rev-parse --verify HEAD` -f ..Dockerfile-java-build --push .``
+
+## Additional Notes
+We are using multi-platform builds (amd64 and arm64) to create separate versions of images that can run on Intel 
+processor architectures and on Mac M1 architecture.
```

Comment:
Referencing M1 directly may be misleading since there are multiple variants, and M2 is on the way. `Apple Silicon` is a more overarching term to refer to Apple's ARM lineup.

```suggestion
processor architectures and on Apple Silicon.
```